src/transports/janus_rabbitmq.c

```diff
@@ -455,7 +461,22 @@ int janus_rabbitmq_init(janus_transport_callbacks *callback, const char *config_
 		rmq_client = g_malloc0(sizeof(janus_rabbitmq_client));
 
 		/* Connect */
-		int result = janus_rabbitmq_connect();
+		uint8_t retry_interval = 5; // Retry interval in seconds
+		int result = -1;
+		while(!rmq_client->destroy && !g_atomic_int_get(&stopping)) {
+			result = janus_rabbitmq_connect();
+			if (result == 0) {
+				break;
+			}
+
+			if (!block_startup_until_connected) {
+				break;
+			}
+
```

Comment:
All these empty lines are unneeded: I think you can make this block more compact without them.